Makefile

```diff
@@ -1,45 +1,43 @@
 .DEFAULT_GOAL		:=help
 SHELL				:=/bin/bash
 
-SERVICE_SHORT=afqts
 KEY_VAULT_PURGE_PROTECTION=false
 ARM_TEMPLATE_TAG=1.1.6
 TERRAFILE_VERSION=0.8
+SERVICE_NAME = apply-for-qualified-teacher-status
```

Comment:
looks like this is overriden by terraform, but it should be set the same i.e. apply-for-qts